integration-tests/resteasy-reactive-rest-client-multipart/pom.xml

```diff
@@ -116,6 +117,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>10</source>
+                    <target>10</target>
+                </configuration>
+            </plugin>
```

Comment:
What's this about? Was added perhaps by IntelliJ?

Reply:
yes :(

Reply:
fixed